### PR TITLE
Minor fixes encountered during automated tool testing

### PR DESCRIPTION
**Pull request by mvdbeek**
Hi there,

these are various fixes I came across while preparing to do automated shed_test installations
using jenkins.

### CODE REVIEW

**Comment by erasche** [comment]
[Jenkins Bot] Can one of the admins sign off on testing this PR? (Security reasons, simply comment "ok to test")

File: packages/package_libpng_1_2/tool_dependencies.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+ <tool_dependency>
+    <package name="libpng" version="1.2.5">
+        <install version="1.0">
+            <actions>
+                <action type="download_by_url">http://download.sourceforge.net/libpng/libpng-1.2.51.tar.gz</action>
+                <action type="shell_command">./configure --prefix=$INSTALL_DIR</action>
```
Comment (bgruening) [inline packages/package_libpng_1_2/tool_dependencies.xml]: @mvdbeek can you use the `<action type="autoconf">` here?

**Comment by bgruening** [comment]
Awesome! Thanks for cleaning up! If you could change this one file and would really like to merge this!

**Comment by bgruening** [comment]
Thanks @mvdbeek!

**Pull request merged**